### ISSUE TITLE
Megadrive doesnt load

### ISSUE THREAD

**Issue by itsdarklikehell** [zshall/program-guide#63]
Megadrive core does not load for me some reason.
i use: 

```
  - number: -3 # Negative numbers indicate line inputs. They don't show up on the channel guide.
    name: Megadrive
    description: Megadrive
    abbr: SEGAMD
    video-game:
      # The video-game engine allows you to run RetroArch cores via Nostalgist
      game: oldtowers.bin # The game library by default is from `retrobrews.github.io`. You can specify a game from there directly.
      console: megadrive # Required. The system to emulate. Allowable values are 'nes', 'snes', 'megadrive', 'gb', 'gbc', or 'gba'.
      # The video game engine doesn't support granular volume adjustment right now, but mute and unmute work.
```
the NES, SNES, GB and GBA cores work fine (even with my wired xbox360 controller. which is awesome!)

also would it be possible to define custom roms/folders to run for instance LSDj.gb on the gameboy or add other libretro systems? like gp32, psp etc. for instance?
im running a [retronas](https://github.com/retronas/retronas) on my network and its filled to the brim with roms/hacks and demoscene, diskmags and stuff. (most of them are TOSEC sets which are also on archive.org)
it would be awesome to be able to run those old artworks as playlists. retroarch itself uses those for rom collections like one could make with retroarch itself (it uses  the .cue .pls. m3u .lpl format playlists for loading for instance multi disks / swapping roms and isos and also for keeping track of playtime/history)
It would be really nice to be able to run [PiggyTracker fka. LitteGPTracker](https://github.com/djdiskmachine/LittleGPTracker) too in one of them.

also another cool channel you could add for example / funzies is:

```
  - number: 106
    name: SceneCity.TV
    description: SceneCity.TV
    abbr: SCNCTYTV
    video:
      volume: true # You can mute the video entirely if you want.
      provider: hls # The HTTP live streaming provider can be utilized if you have such a video stream.
      src: https://fra1-edge2.scenecity.net/origin3/demos/avc-auto_llhls.m3u8
      hlsBugFix: false # I've noticed that some HLS streams play automatically and if we attempt to play them again the video won't load. This is a workaround for that.
```

**Comment by zshall** [comment]
I've reproduced the problem; it looks like Nostalgist's Megadrive core is hosted on a CDN and isn't being loaded. I'll check if there's an update available first.

**Comment by zshall** [comment]
Looks like it may be related to the core name. Investigating further.

**Comment by zshall** [comment]
One problem I've had with supporting newer systems is keyboard mappings. I'm not sure where all the extra buttons are going to go and as of now the `video-game` engine doesn't support key remapping, so I want to make sure I can get that working first. When you say you've gotten an Xbox 360 controller to work with TVS I'd be curious to know how you did it actually; did you use a program like JoyToKey or is there native support in the libretro library that I'm not familiar with? I'll write that into the documentation.

As far as bringing your own game library on a NAS, this is possible given you can run a web server that has CORS enabled (`Access-Control-Allow-Origin: *`) and your ROMs are available via the web server on your LAN. Just use the full URL:

~~~
- number: -4 # Negative numbers indicate line inputs. They don't show up on the channel guide.
  name: Local game example
  description: Game running on a NAS
  abbr: LGAME
  video-game:
    # The video-game engine allows you to run RetroArch cores via Nostalgist
    game: http://your-nas/games/nes/Game%20Name.zip
    console: nes
~~~

Playlists sound like an interesting idea.

* [ ] We'd need a file format that TVS could read that had the list of games and their URLs, or just a list of games and a base URL.
* [ ] We could have a next / previous button for games and a random button

**Comment by itsdarklikehell** [comment]
Lovely work m8.
Those pesky europeans with their wierd naming-schemes.

as for the xbox controller thing is probably allso because i run retropie/emulationstation/retroarch stuff and afaik RetroArch does support the wired xbox controller profiles by default. 
as far as wireless controllers goes (i allso have an old ps3 controller which I sometimes use) you obviously first need to setup a bleutooth connection with the controller and RetroArch should be able to use that too. I havent tested that yet because its battery is drained atm. so I connected it straight to usb (it then becomes a native generic usb controller) which I tested and works too.
